### ISSUE TITLE
AccessibilityInsight: Inspect: IsExpandCollapsePattemAvailable property for ListViewGroup should be false when setting ListViewGroup's CollapsedState is default

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#5981]
* .NET Core Version: 
.Net 6.0.100-rtm.21513.45 SDK build

* Have you experienced this same bug with .NET Framework?: 
No, there is no CollapsedState property for ListViewGroup

**Minimal repro:**
1. Create a winforms application with ListView added
2. Add a ListViewGroup, and set its CollapsedState to Default
3. Add several items and then set items' group to this added group
4. Build and run app, then use Inspect to check this ListViewGroup

or use following attached application directly for ListViewGroup3
[WinFormsApp13.zip](https://github.com/dotnet/winforms/files/7352020/WinFormsApp13.zip)

**Actual Result:**
In Inspect, the IsExpandCollapsePattemAvailable property should be false, not true. Because when setting ListViewGroup's CollapsedState is default, the listviewGroup is expand state by default, but it doesn't support user uses mouse or keyboard to collapse this ListViewGroup.
![image](https://user-images.githubusercontent.com/26474449/137456849-68a03389-19c2-4827-932a-daa337c70614.png)

**Expected Result:**
In Inspect, the IsExpandCollapsePattemAvailable property is false when the ListViewGroup is not allowed to collapse or expand.
